### PR TITLE
Ubuntu16 build has been fixed

### PR DESCRIPTION
**Pull request by andy-mishechkin**
According to https://github.com/actions/virtual-environments/issues/353, dotnetcore-sdk.sh has been modified.
Addition of extra repository 'microsoft-ubuntu-bionic-prod' has been removed from this script (right microsoft package already is added through repos.sh script), because it impacted to PowerShell Core installation and build has been broken
After this modification Ubuntu16 build is successful.

### CODE REVIEW

**Comment by miketimofeev** [comment]
@andy-mishechkin do we need the same for ubuntu18 dotnet installation script just to be consistent?

**Comment by andy-mishechkin** [comment]
I didn't investigate Ubuntu 18 deployment yet, and I need some time for this

File: images/linux/scripts/installers/1604/dotnetcore-sdk.sh [hunk 1]
```diff
@@ -34,11 +34,6 @@ for latest_package in ${LATEST_DOTNET_PACKAGES[@]}; do
     echo "Determing if .NET Core ($latest_package) is installed"
     if ! IsInstalled $latest_package; then
         echo "Could not find .NET Core ($latest_package), installing..."
-        curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
-        mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
-        sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main" > /etc/apt/sources.list.d/dotnetdev.list'
-        apt-get install apt-transport-https
```
Comment (sergey-akhalkov) [inline images/linux/scripts/installers/1604/dotnetcore-sdk.sh]: Could you please confirm we don't really need `apt-transport-https` package?

File: images/linux/scripts/installers/1604/dotnetcore-sdk.sh [hunk 2]
```diff
@@ -34,11 +34,6 @@ for latest_package in ${LATEST_DOTNET_PACKAGES[@]}; do
...
-        apt-get update
         apt-get install $latest_package -y
```
Comment (sergey-akhalkov) [inline images/linux/scripts/installers/1604/dotnetcore-sdk.sh]: Could you please also confirm that all packages in the LATEST_DOTNET_PACKAGES are publicly available (no need to and any new packages sources / etc.) and can be installed without any issues?

**Comment by al-cheb** [comment]
> @andy-mishechkin do we need the same for ubuntu18 dotnet installation script just to be consistent?

You should change docker-moby.sh script for Ubuntu 18.04 to preserve the consistence.

File: images/linux/scripts/installers/1604/dotnetcore-sdk.sh (hunk 1 above)
Comment (andy-mishechkin) [reply images/linux/scripts/installers/1604/dotnetcore-sdk.sh]: This package also will be installed in kubernetes-tools.sh
I think we don't need install it twice in Ubuntu16 build

File: images/linux/scripts/installers/1604/dotnetcore-sdk.sh (hunk 2 above)
Comment (andy-mishechkin) [reply images/linux/scripts/installers/1604/dotnetcore-sdk.sh]: Confirmed
The last version of .NET Core is 3.1 (https://dotnet.microsoft.com/download)
It presents in LATEST_DOTNET_PACKAGES

File: images/linux/scripts/installers/1604/dotnetcore-sdk.sh (hunk 1 above)
Comment (sergey-akhalkov) [reply images/linux/scripts/installers/1604/dotnetcore-sdk.sh]: Does it matter when this is installed - earlier or later? Are we using it somehow during the image generation?

**Pull request merged**